src/extension.ts

```diff
@@ -121,27 +121,32 @@ async function updateClusters(provider: extensionApi.Provider, containers: exten
       const lifecycle: extensionApi.ProviderConnectionLifecycle = {
         start: async (): Promise<void> => {
           try {
-            const env = { ...process.env };
-            env.PATH = getMinikubePath();
-            await extensionApi.process.exec(minikubeCli, ['start', '--profile', cluster.name], { env });
+            await extensionApi.process.exec(minikubeCli, ['start', '--profile', cluster.name], {
+              env: {
+                PATH: getMinikubePath(),
```

Comment:
not related to this PR but it's strange sometimes we have both MINIKUBE_HOME and sometimes only the PATH


also should we call a custom utility method that will insert minikubePath and MinikubeHome instead of doing that at several places

Reply:
I think this is the core issue actually, thanks for pointing that out, the code should be centralized, and we should have a single function that create the envs required